packages/shared/lib/core/wallet/utils/formatBestMatchTokenAmount.ts

```diff
@@ -0,0 +1,10 @@
+import { ITokenMetadata } from '../interfaces'
+import { formatDefaultTokenAmount } from './formatDefaultTokenAmount'
+
+export function formatBestMatchTokenAmount(amount: number, tokenMetadata: ITokenMetadata): string {
+    if (tokenMetadata.useMetricPrefix) {
+        return amount + ' ' + tokenMetadata.unit
```

Comment:
We may need to add a call to `formatNumber(amount)` here so that the proper decimal separator is used for different locales.